lib/msal-react-native-android-poc/android/src/main/java/com/reactlibrary/MSALModule.java

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+/**
+ * MSALModule.java functions as a wrapper around MSAL for Android's signing in, signing out, and acquiring tokens functionality.
+ * It is a proof of concept, focusing on single account mode and AAD.
+ */
+
+package com.reactlibrary;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+import com.facebook.react.bridge.WritableNativeMap;
+import com.microsoft.identity.client.*;
+import com.microsoft.identity.client.exception.MsalException;
+
+import static com.facebook.react.views.textinput.ReactTextInputManager.TAG;
+
+//import DEFAULT_SCOPES from Constants.java
+import static com.reactlibrary.Constants.DEFAULT_SCOPES;
+
+import java.util.Arrays;
+
+public class MSALModule extends ReactContextBaseJavaModule {
+
+    private final ReactApplicationContext reactContext;
+    private ISingleAccountPublicClientApplication publicClientApplication;
+
+    public MSALModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+        this.reactContext = reactContext;
+
+        try{
+            //initialize publicClientApplication with config file, located in main/java/res/raw/auth_config_single_account.json
+            publicClientApplication = PublicClientApplication.createSingleAccountPublicClientApplication(
+                reactContext,
+                reactContext.getResources().getIdentifier("auth_config_single_account", "raw", reactContext.getPackageName()));
+        } catch (Exception e) {
+            //will handle this
+            Log.d(TAG, "Something wrong with initialization of publicClientApplication: " + e.toString());
+        }
+    }
+
+     /**
+   * getName(): MSAL is the name used when importing the module from native modules
+   */
+    @Override
+    public String getName() {
+        return "MSAL";
```

Comment:
Move "MSAL" to constants file and reference here

Reply:
This, and all of the map keys are now in the constants file. All of the constants are utilized in the sample app now.